doc/source/whatsnew/v0.17.0.txt

```diff
@@ -613,3 +613,4 @@ Bug Fixes
 
 - Bug in ``pd.DataFrame`` when constructing an empty DataFrame with a string dtype (:issue:`9428`)
 - Bug in ``pd.unique`` for arrays with the ``datetime64`` or ``timedelta64`` dtype that meant an array with object dtype was returned instead the original dtype (:issue: `9431`)
```

Comment:
dont reference internal functions
just say that bug in groupy with NaT were computing incorrectly 
someone who wants more detail can always drill down